cpp/dolfinx/common/IndexMap.h

```diff
@@ -161,11 +165,10 @@ class IndexMap
   std::int64_t local_to_global(std::int32_t local_index) const
```

Comment:
Yes, I think so. It would be fairly simple to use `_local_range` and `ghosts` in-line instead.
I'll create a new PR removing it, and a few more cleanups.